packages/esm-patient-chart-app/src/visit/visits-widget/visit-detail-overview.component.tsx

```diff
@@ -41,7 +41,7 @@ function VisitDetailOverviewComponent({ patientUuid }: VisitOverviewComponentPro
             {t('visitSummaries', 'Visit summaries')}
           </Tab>
           <Tab className={styles.tab} id="all-encounters-tab">
-            {t('allVisits', 'All visits')}
+            {t('allEncounters', 'All encounters')}
```

Comment:
Why do you have this change in this PR, it looks like you branch off the other branch which has this work, every new branch should always be made from main to avoid carrying changes from other branches